caddyhttp/browse/setup.go

```diff
@@ -426,7 +426,9 @@ footer {
 								<span class="name">{{html .Name}}</span>
 							</a>
 						</td>
-						{{- if .IsDir}}
+						{{- if .IsSymlink }}
+						<td data-order="-1">symbolic link</td>
```

Comment:
Instead of showing the words "symbolic link" under the "Size" column, how about we introduce new icons for symbolic link to file and symbolic link to directory?